TWLight/templates/header_partial_b4.html

```diff
@@ -61,9 +61,31 @@
       <i class="navbar-toggler-icon fa fa-arrow-down"></i>
     </button>
 
-    <div class="col-lg-4 col-md-8 col-sm-3 col-12 offset-md-4 offset-lg-0">
+    <div class="col-lg-5 col-md-8 col-sm-3 col-12 offset-md-4 offset-lg-0">
       <div id="navbarTogglerTWL" class="collapse navbar-collapse justify-content-lg-end justify-content-md-end">
         <ul class="navbar-nav">
+          <li class="mobile-menu align-middle nav-item">
+            <form action="/i18n/setlang/?next=/" method="post" class="form-inline">
+                {% csrf_token %}
+                {% if form_show_errors %}
+                  {% include "bootstrap4/errors.html" %}
+                {% endif %}
+                <div class="dropdown">
+                  <button class="btn btn-link dropdown-toggle nav-link nav-links language-select-button" type="button" id="language-dropdown" data-toggle="dropdown"
+                          aria-haspopup="true" aria-expanded="false">
+                          {% for code, name in LANGUAGES %}
```

Comment:
I'd like to DRY this up a bit by not looping twice. 
The only problem is we've added additional locales not supported by Django's built-in helper function get_language_info so I'm not sure how else to grab localized names without looping through all the options and checking for equality. 

Any ideas or thoughts?

Reply:
You are right, we can do this without looping. Try https://docs.djangoproject.com/en/4.2/topics/i18n/translation/#get-language-info.

Reply:
Yeah, I had tried that method and was getting an exception locally for locales that aren't supported out of the box by Django. 

For example, Bikol Central returns the following error: 
![Screenshot 2024-05-22 at 6 13 45 AM](https://github.com/WikipediaLibrary/TWLight/assets/30132257/d58d4e03-dbf9-4bd4-af1a-93edeea7ddde)

Reply:
Based on that, I think we can leave this as-is. I hope Django will one day catch-up on what languages they support.